docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -37,35 +60,81 @@ gcloud compute scp <master-node-instance>:/var/run/kubernetes/client-ca.crt /tmp
+
 At the moment this doc is written, kube-proxy does not support multi-tenancy yet, and it won't be deployed in the cluster. After kube-proxy issue has been fixed (we will allocate resource to cope with it soon), probably we will also need to copy over kube-proxy related secret and artifacts.
 
 As a temporary measure to accommodate Flannel pods to access api server through the kubernetes service IP (the notorious 10.0.0.1) at the absense of kube-proxy, we need to place a bit trick by running below command __at the worker node__, assuming the kupe-apiserver is listening at https://10.138.0.19:6443 (please substitute with the proper value of your own cluster):
+
 ```bash
 sudo iptables -t nat -A OUTPUT -p tcp -d 10.0.0.1 --dport 443 -j DNAT --to-destination 10.138.0.19:6443
+sudo iptables -t nat -L (for verification)
 ```
 
 NOTE: you need to re-run the above command once you restart the machine.
 
-Please be advised that this is a temporary quirk only; after we have the full service support by proper kube-proxy or other means, we don't need it any more.
+Please be advised that this is a temporary quick only; after we have the full service support by proper kube-proxy or other means, we don't need it any more.
 
-4. start the worker node and register into cluster
+Then you should see pod 'pod/kube-flannel-ds-xxxxx' is in Running state after you run command "./cluster/kubectl.sh get all --all-namespaces". 
 
-First Make sure the following directories in the worker node are empty. If not, clean them up.
-```
-/opt/cni/bin/
-/etc/cni/net.d/
+6. Test whether the ngnix application can be deployed successfully
+
+NOTE: You need first run the following command to create clusterrolebinding 'system-node-role-bound' to bind the group 'system:nodes' to clusterrole 'system:node' so that the master node has corresponding permission to get secret for every namespace and transfer the secret to worker node during pod creation.
+
+```bash
+./cluster/kubectl.sh create clusterrolebinding system-node-role-bound --clusterrole=system:node --group=system:nodes
+./cluster/kubectl.sh get clusterrolebinding/system-node-role-bound -o yaml
 ```
 
-Then at worker node, run following commands:
+Then you can run container pods for nginx now and see all pods should be in Running state.
 ```bash
-export ARKTOS_NO_CNI_PREINSTALLED=y
-export KUBELET_IP=<worker-ip>
-./hack/arktos-worker-up.sh
+./cluster/kubectl.sh run nginx --image=nginx --replicas=2
+./cluster/kubectl.sh get all -n default
 ```
 
-After the script returns, go to master node terminal and run command "[arktos_repo]/cluster/kubectl.sh get nodes", you should see the work node is displayed and its status should be "Ready".
 
-5. label worker node as vm runtime capable (optional)
+7. Label worker node as vm runtime capable (optional)
 
 If you would like to allow this work node to run VM-based pods, please run below command at the master console:
 ```bash
```

Comment:
Section below also need be updated:
run the command below:
```
./cluster/kubectl.sh run nginx --image=nginx --replicas=2
./cluster/kubectl.sh get pod -o wide
```
get errors:
```
sonyafenge@doctests-worknode:~/go/src/arktos$ ./cluster/kubectl.sh run nginx --image=nginx --replicas=2
kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
Error from server (AlreadyExists): deployments.apps "nginx" already exists
```

Reply:
This section need  to be removed: "How to set up multiple partitioned apiservers". we don't support multi-apiserver anymore.

Reply:
This section need more information for how to run a VM pod or removed from this doc: "Label worker node as vm runtime capable"

Reply:
Add the command to delete ngnix deployment in the beginning. Thanks!

Reply:
Sonya suggestion:
2.	“Label worker node as vm runtime capable”, add more information for this section or remove this section. At least from current description, I am not sure how to  run a VM pod. @Yunwen Bai may know more information for this section.


I remove this step 7 (optional).